src/tunacode/core/agents/resume/prune.py

```
@@ -234,40 +211,38 @@ def prune_old_tool_outputs(
 
-    # Phase 2: Confirm pruning boundary
-    # Early exit: nothing old enough to prune
-    if prune_start_index < 0:
-        return (messages, 0)
 
-    # Phase 3: Calculate potential savings
-    parts_to_prune = tool_parts[prune_start_index:]
+def _has_sufficient_savings(
+    parts_to_prune: list[tuple[int, int, Any, int | None]],
+    minimum_threshold: int,
+    stop_estimating: bool,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)